packages/service-library/src/servicelib/fastapi/http_client_thin.py

```diff
@@ -61,18 +119,16 @@ def retry_on_errors(
     """
     assert asyncio.iscoroutinefunction(request_func)
 
-    RETRY_ERRORS = (ConnectError, PoolTimeout)
-
     @functools.wraps(request_func)
     async def request_wrapper(zelf: "BaseThinClient", *args, **kwargs) -> Response:
```

Comment:
This is a technique we used but is far from ideal that is way i never wanted to put it in a common repo... I would advice against using it everywhere.  There are cleaner ways to do this... I have